gcloud-java-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBlobSnippets.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.example.storage.snippets;
+package com.google.cloud.examples.storage.snippets;
```

Comment:
> We've been trying to use com.example.blah (RFC 2606) for everything

Can you give me more details? Are you suggesting that the package should be named `com.example`? This sounds strange to me.